internal/server/routes.go

```diff
@@ -155,7 +155,6 @@ func post[Req, Res any](a *API, r *gin.RouterGroup, path string, handler ReqResH
 	fullPathStr := fullPath(r, path)
 
 	r.POST(path, func(c *gin.Context) {
-		c.Set("path", fullPathStr)
```

Comment:
These are the calls I was removing

Reply:
used in

		a.t.Event(c, "error", Properties{
			"code": resp.Code,
			"path": c.GetString("path"),
		})

Reply:
Ahhh, I did not realize there were multiple ways of getting values out of context!  I was searching for `c.Get("path")`.

That's good to know.

Reply:
Good catch @ssoroka !